exit-what-1.md

```diff
@@ -3,6 +3,7 @@ layout: default
 title: Exit What 1
 ---
 
-# Exit what 1
+# Things to know:
```

Comment:
Can we make this heading more descriptive? Maybe something like "Timeline for requesting your health information"?